### DISCUSSION TITLE
upload image while posting an article in backend is not working

### DISCUSSION THREAD

**Discussion by Rakesh653** [nasirkhan/laravel-starter#410] [General]
While uploading an image, filemanager window gets open.
But nothing happen on selecting a picture from it.

**Comment by nasirkhan** [comment]
related to https://github.com/nasirkhan/laravel-starter/issues/411

**Comment by nasirkhan** [comment]
This issue has been resolved in the new release.

**Comment by valonsoft** [comment]
This issue is still very much present
